mods/ts/rules/defaults.yaml

```diff
@@ -1160,30 +1156,37 @@
 			VerticalTopOffset: 768
 	EditorTilesetFilter:
 		Categories: Wall
+	Gate:
+		OpeningSound: gateup1.aud
+		ClosingSound: gatedwn1.aud
+		BlocksProjectilesHeight: 768
+		PauseOnCondition: empdisable
 
 ^Gate_A:
 	Inherits: ^Gate
 	Inherits@shape: ^3x1Shape
 	Health:
-	Gate:
+	Building:
 		Dimensions: 3,1
 		Footprint: xxx
 	WithGateSpriteBody:
 		WallConnections: -1,0, 3,0
 	LineBuildNode:
 		Connections: -1,0, 1,0
+	Gate:
```

Comment:
`^Gate_A` already inherits `^Gate`, which has the `Gate` trait, so this is redundant.